docs/notebooks/tutorials/QFM_surface.ipynb

```diff
@@ -0,0 +1,783 @@
+{
```

Comment:
<ul><li>I think \argmin is not a proper command. Maybe change it to \arg\min_S ?</li><li>I would prefer double Fourier series instead of "two Fourier series"</li><li>Can you also mention here that the surface has to enclose some given toroidal flux too</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaControl/DESC/pull/1264/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>